.github/workflows/ci_tests.yaml

```diff
@@ -82,11 +82,51 @@ jobs:
 
     - name: E2E Tests
       run: |
-        make deploy-ci NAMESPACE=mygatekeeper IMG=localhost:5000/gatekeeper-operator:$GITHUB_SHA
-        kubectl -n mygatekeeper wait deployment/gatekeeper-operator-controller --for condition=Available --timeout=90s
-        kubectl -n mygatekeeper logs deployment/gatekeeper-operator-controller -c manager -f > operator.log &
-        make test-e2e NAMESPACE=mygatekeeper
-        kubectl delete --wait namespace mygatekeeper
+        make deploy-ci NAMESPACE=gatekeeper-system IMG=localhost:5000/gatekeeper-operator:$GITHUB_SHA
+        kubectl -n gatekeeper-system wait deployment/gatekeeper-operator-controller --for condition=Available --timeout=90s
+        kubectl -n gatekeeper-system logs deployment/gatekeeper-operator-controller -c manager -f > operator.log &
+        make test-e2e NAMESPACE=gatekeeper-system
```

Comment:
Why the changes to `gatekeeper-system`? I think that's the default? I might prefer we use `mygatekeeper` for all or some tests so that we know that deploying to something other than the default namespace still works?

Reply:
In makefile, gatekeeper-system is default so everytime when I test on my local, I had to change to gatekeeper-system. I just picked gatekeeper-system but I can change to mygatekeeper

Reply:
I had that problem also, but it was because the E2E function was being called from another wrapping function, so I think the arguments weren't getting passed down properly after I upgraded Ginkgo. Did you try it after rebasing on my update? I think it should work now.

Reply:
oh yes actually before work well, after rebase didn't work.

Reply:
I just tried it locally, and it's passing consistently. The one inconsistency I found was from the Pod Security blocking the deployment, but I noticed you have that set in the other PR, so maybe that'd resolve it?

Reply:
yes... seccomProfile it actually solve issue

Reply:
But  I am doubt it... because it didn't happen before.  I am not sure why this happen

Reply:
My suspicion is that it's because we bumped the Kubernetes version from `v1.21` to `v1.28`, but I'm not sure.

Reply:
hm.......  but last week was ok